it/controllers/CalculationControllerISpec.scala

```diff
@@ -450,21 +656,103 @@ class CalculationControllerISpec extends IntegrationSpecCommonBase {
+        parsedBody.select("#main-content .govuk-body").get(0).text() shouldBe "Paying part of your VAT bill will reduce further penalties."
+        parsedBody.select("#main-content .govuk-body").get(1).text() shouldBe "Penalties and interest will show as estimates if HMRC has not been given enough information to calculate the final amounts."
+        parsedBody.select("#main-content h2").text() shouldBe "Estimates"
+        parsedBody.select("#main-content a").get(0).text() shouldBe "Return to VAT penalties and appeals"
+        parsedBody.select("#main-content a").get(0).attr("href") shouldBe "/penalties"
+      }
+
+      "the user has specified a valid penalty ID (parses decimals correctly)" in new Setup(true) {
+        returnPenaltyDetailsStub(samplePenaltyDetails)
+        val request = controller.onPageLoad("12345678901234", false)(fakeRequest)
+        status(request) shouldBe Status.OK
+        val parsedBody = Jsoup.parse(contentAsString(request))
+        parsedBody.select("#main-content h1").first().ownText() shouldBe "Late payment penalty"
+        parsedBody.select("#main-content header p").first.text() shouldBe "The period dates are 1 January 2021 to 1 February 2021"
+        parsedBody.select("#main-content header p span").first.text() shouldBe "The period dates are"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(0).select("dt").text() shouldBe "Penalty amount (estimate)"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(0).select("dd").text() shouldBe "£400.00"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(1).select("dt").text() shouldBe "Calculation"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(1).select("dd").text() shouldBe "2% of £123.00 (VAT amount unpaid on 23 March 2021) + 2% of £123.00 (VAT amount unpaid on 7 April 2021)"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(2).select("dt").text() shouldBe "Amount received"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(2).select("dd").text() shouldBe "£277.00"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(3).select("dt").text() shouldBe "Amount left to pay"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(3).select("dd").text() shouldBe "£123.00"
+        parsedBody.select("#main-content div .govuk-warning-text").text() shouldBe "! This penalty will rise to £246.00 (a further 2% of the unpaid VAT) if you do not make a VAT payment by 7 April 2021."
+        parsedBody.select("#main-content .govuk-body").get(0).text() shouldBe "Paying part of your VAT bill will reduce further penalties."
+        parsedBody.select("#main-content .govuk-body").get(1).text() shouldBe "Penalties and interest will show as estimates if HMRC has not been given enough information to calculate the final amounts."
+        parsedBody.select("#main-content h2").text() shouldBe "Estimates"
```

Comment:
You can remove this because the penalty would be posted, unless you want to keep it as accruing